### PR TITLE
Add Single VM support in Connections Overview

### PR DESCRIPTION
**Pull request by andrewki-msft**
`Connections Overview` now appears in Single VM workbooks gallery. Additional helper text and support for Virtual Machine resource has been added to the workbook.

Single VM
image

AtScale
image

### CODE REVIEW

**Comment by nasundar** [comment]
Suggestion:

Move this up, its not discover able with a large grid.

image

File: Workbooks/Virtual Machines - Network Dependencies/Connections Overview/Connections Overview.workbook [hunk 1]
```diff
@@ -39,6 +27,142 @@
...
+          "{Workspace}"
+        ],
+        "parameters": [
+          {
+            "id": "2debcfba-e410-4300-9886-78069cef42e0",
+            "version": "KqlParameterItem/1.0",
+            "name": "_A_ComputerName",
+            "type": 1,
+            "isRequired": false,
+            "query": "print(iff(\"{ComputerName}\" == \"undefined\", \"\", \"<h2 style='color:#777;font-weight:normal;margin-top:0;padding-top:0;font-size:1rem;'>{ComputerName}</h2>\"));",
```
[the remark this answers is not in the document]
Comment (andrewki-msft) [reply Workbooks/Virtual Machines - Network Dependencies/Connections Overview/Connections Overview.workbook]: @gardnerjr May I keep using custom CSS styles in my markdowns or should I deprecate using them (i.e. any plans to remove CSS support?)

**Comment by andrewki-msft** [comment]
> Any idea this workbook shows 'Default items'?

Fixed, we need to make sure the `isDefault` field is not present in the dropdown and hard-code the default value

**Comment by andrewki-msft** [comment]
> Move this up, its not discover able with a large grid.

Done

File: Workbooks/Virtual Machines - Network Dependencies/Connections Overview/Connections Overview.workbook [hunk 2]
```diff
@@ -424,51 +636,90 @@
...
         "query": "",
         "crossComponentResources": [
-          "{Workspaces}"
+          "{Workspace}"
         ],
         "parameters": [
+          {
+            "id": "5e335a1b-7f99-4647-854a-d7b5cb489bb2",
+            "version": "KqlParameterItem/1.0",
+            "name": "QC",
```
[the remark this answers is not in the document]
Comment (andrewki-msft) [reply Workbooks/Virtual Machines - Network Dependencies/Connections Overview/Connections Overview.workbook]: Done

**Comment by gardnerjr** [comment]
We don’t have any plans ourselves on removing css/etc. just remember whatever you do needs to work on both light and dark themes, so be careful!

**Pull request merged**